sources/packages/web/src/types/contracts/ConfigContract.ts

```diff
@@ -9,6 +9,13 @@ export enum ClientIdType {
   SupportingUsers = "supportingUsers",
 }
 
+export enum ClientTypeBaseRoute {
+  Student = "student",

```

Comment:
Please change it to _students_ to follow the same from others.

Reply:
oh..i thought  i changed, updating